metafix/src/main/java/org/metafacture/metafix/Metafix.java

```diff
@@ -313,22 +315,36 @@ public void startEntity(final String name) {
             throw new IllegalArgumentException("Entity name must not be null.");
         }
 
+        ++entityCount;
+        if (maxEntityCountExceeded()) {
+            LOG.debug("Maximum number of entities exceeded: {}/{}", entityCount, MAX_ENTITY_COUNT);
```

Comment:
We could add a field, and only log when it's set for the first time, but since this is on `debug` level I think it's fine. Also, it might be useful to see how many entities were actually skipped.